### DISCUSSION TITLE
How can I disallow user to change email

### DISCUSSION THREAD

**Discussion by NoddleAres** [fastapi-users/fastapi-users#1260] [Q&A]
Currently user can change their own email through patch. I was wondering if I can disallow user from changing their email after they are registered

**Comment by frankie567** [comment]
What you can do is to define an update schema without the `email` property. Something like this:

```py
from fastapi_users import schemas

class UserUpdate(schemas.CreateUpdateDictModel):
    password: Optional[str] = None
    is_active: Optional[bool] = None
    is_superuser: Optional[bool] = None
    is_verified: Optional[bool] = None
```
